### PR TITLE
Create contact form

### PR DESCRIPTION
**Pull request by ntzwilly**
# Portfolio: add a contact form and finish the mobile version

## Project requirements:

- Build the contact form in the footer:

   Make sure that:
    - Name field accepts up to 30 chars.
    - Email field accepts only emails.
    - Message field accepts up to 500 chars.
    - All fields have placeholder text.
    - All fields have the correct type.
    - A user cannot submit the form if one or more of the fields is empty.
 
- Collect data by using the Formspree service.
- In order to layout the elements on the page, use Flexbox.
- No need to implement any functionality that requires JavaScript.
- Stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma.
Implement the button interactions (enable, hover, etc.).
- Use pair programming for this project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -303,6 +303,23 @@ <h1 class="title-item">Tonic</h1>
...
+      <p class="bottom-text">
+        <a href="#">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </a>
+      </p>
+      <form class="form" action="https://formspree.io/f/xwkabeng" method="post">
+        <input class="text-input" type="text" maxlength="30" placeholder="Full name" required><br>
+        <input class="email-input" type="email" placeholder="Email address" required>
```
Comment (ElijahTobs) [inline index.html:317]: - [ ] You should add the `name` attribute on the each of the input tags as formspree only save fields with name attribute.
image

File: README.md [hunk 2]
```diff
@@ -2,14 +2,10 @@
 
 # Project 2: Mobile version
 
-> This second project consists to create  mobile website sections where I list my portfolio projects and the About me section.
+> In this third project, we create the contact form at the bottom of the mobile version
+
+![form_screenshot](https://user-images.githubusercontent.com/9049260/121239480-95942e80-c899-11eb-83c2-8489ef9c45ea.png)
```
Comment (ElijahTobs) [inline README.md:7]: - [ ] _[Optional]_ The link to your live demo is not working. Kindly fix it

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi Team 👋

You've done really well so far! 👍 

## Major Highlights 💯 

Your README is professional and you've followed the Github flow rules. ✔️ 
You added validations for the input fields ✔️ 
You've got good commit messages ✔️ 

However, there are some changes that you are required to make before this project can be approved.

## Required Changes ♻️


_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

Check for changes marked as Optional in the inline comments below.


#### If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

### Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#), however, if you feel this review is unfair, you can request a second opinion using this [form](https://airtable.com/shrQAqnBwek5a0O0s).

## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: index.html (hunk 1 above)
Comment (ntzwilly) [reply index.html]: I added a name attribute on each input

File: README.md (hunk 2 above)
Comment (ntzwilly) [reply README.md:7]: The link is now working

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html]: Well resolved this!

File: README.md (hunk 2 above)
Comment (DeliceLydia) [reply README.md:7]: The live demo is working perfectly. Well done!

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved🎉 🎉 

Hi @ntzwilly, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**